src/App/Repository/ConfigRepository.php

```diff
@@ -104,6 +119,23 @@ public function update(Entity $entity)
 		}
 	}
 
+	private function insertOrUpdate($group, $key, $value)
+	{
+		$value = json_encode($value);
+
+		try {
+			DB::insert('config', array('group_name', 'config_key', 'config_value'))
+				->values(array($group, $key, $value))
+				->execute($this->db);
+		} catch (\Database_Exception $e) {
```

Comment:
> Really not sure about this, but I think it would go away once we have entities to manage certain configs.

Its gross but taken directly from the Kohana Database Config driver. I was going for minimal changes, rather than sensible code. Easy to improve though